ui/src/components/header/ServiceInstanceDetails.js

```diff
@@ -31,27 +42,173 @@ const StyledAnchor = styled.a`
+                                                enableTitle={false}
+                                            />
+                                        )}
+                                        triggerOn='click'
+                                    >
+                                        <StyledAnchorDiv>
+                                            {
+                                                TOTAL_HEALTHY_DYNAMIC_INSTANCES_DESC
+                                            }
+                                        </StyledAnchorDiv>
+                                    </Menu>
+                                </SectionHeader>
+                            </LabelDiv>
+                        </SectionDiv>
+                    </span>
+                )}
+
+                {categoryType === 'Static' && (
+                    <span>
+                        <SectionDiv>
+                            <ValueDiv>
+                                {categoryType === 'Static'
+                                    ? instanceDetailsMeta.workloadMeta
+                                          .totalStatic
+                                    : 'NA'}
+                            </ValueDiv>
+                            <LabelDiv>
+                                {TOTAL_STATIC_INSTANCES_LABEL}
+                                <SectionHeader>
+                                    <Menu
+                                        placement='bottom-end'
+                                        padding-left='10px'
+                                        trigger={({
+                                            getTriggerProps,
+                                            triggerRef,
+                                        }) => (
+                                            <Icon
+                                                icon={'help-circle'}
+                                                {...getTriggerProps({
+                                                    innerRef: triggerRef,
+                                                })}
+                                                isLink
+                                                size={'15px'}
+                                                color={colors.graphBlue}
+                                                enableTitle={false}
+                                            />
+                                        )}
+                                        triggerOn='click'
+                                    >
+                                        <StyledAnchorDiv>
+                                            {TOTAL_STATIC_INSTANCES_DESC}
+                                        </StyledAnchorDiv>
+                                    </Menu>
+                                </SectionHeader>
+                            </LabelDiv>
+                        </SectionDiv>
+                    </span>
+                )}
```

Comment:
we have total instances (static+dynamic) data as well. we can display that as well

Reply:
not needed for first iteration.